### PR TITLE
#356 minimize the number of outputs  (constraints and lostEquipments)

### LINKED ISSUE #356: [Outputs] Minimize the number of outputs of SA
platform (PowSybl)  needs only the aggregated results and some output files (contraints, lost equipments...) and not the entire output iidm. This last could be dumped only in debug mode

### PR DESCRIPTION
**Pull request by rosiereflo**
and disable iidm dump in Release
close #356

**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [x] unit tests and non regression tests were added (update of inputs, outputs, SA, algos)
- [x] main documentation was updated (update of input/output file)
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

File: tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling/outputIIDM.xml [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
-            <iidm:vscConverterStation id="VSCStation1" name="VSCS1" voltageRegulatorOn="true" lossFactor="0" voltageSetpoint="400" reactivePowerSetpoint="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="20.031018988425391">
+            <iidm:vscConverterStation id="VSCStation1" name="VSCS1" voltageRegulatorOn="true" lossFactor="0" voltageSetpoint="399.97780496606896" reactivePowerSetpoint="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="20.031018988425377">
                 <iidm:reactiveCapabilityCurve>
                     <iidm:point p="-1011" minQ="-300" maxQ="300"/>
                     <iidm:point p="0" minQ="-600" maxQ="400"/>
                     <iidm:point p="1011" minQ="-300" maxQ="300"/>
                 </iidm:reactiveCapabilityCurve>
             </iidm:vscConverterStation>
-            <iidm:lccConverterStation id="LCCStation1" name="S1" lossFactor="0" powerFactor="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="-0"/>
+            <iidm:lccConverterStation id="LCCStation1" name="S1" lossFactor="0" powerFactor="-1" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="1.2246467991473532e-14"/>
```
Comment (marcochiaramello) [inline tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling/outputIIDM.xml:34]: why the power factor is changing?

File: tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling_diagrams/outputIIDM.xml [hunk 2]
```diff
@@ -24,14 +24,14 @@
...
-            <iidm:vscConverterStation id="VSCStation1" name="VSCS1" voltageRegulatorOn="true" lossFactor="0" voltageSetpoint="400" reactivePowerSetpoint="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="20.031018988425391">
+            <iidm:vscConverterStation id="VSCStation1" name="VSCS1" voltageRegulatorOn="true" lossFactor="0" voltageSetpoint="399.97780496606896" reactivePowerSetpoint="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="20.031018988425377">
                 <iidm:reactiveCapabilityCurve>
                     <iidm:point p="-1011" minQ="-300" maxQ="300"/>
                     <iidm:point p="0" minQ="-600" maxQ="400"/>
                     <iidm:point p="1011" minQ="-300" maxQ="300"/>
                 </iidm:reactiveCapabilityCurve>
             </iidm:vscConverterStation>
-            <iidm:lccConverterStation id="LCCStation1" name="S1" lossFactor="0" powerFactor="0.90000000000000002" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="-0"/>
+            <iidm:lccConverterStation id="LCCStation1" name="S1" lossFactor="0" powerFactor="-1" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="1.2246467991473532e-14"/>
```
Comment (marcochiaramello) [inline tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling_diagrams/outputIIDM.xml:34]: same question as above

Comment (marcochiaramello) [inline tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling_diagrams/outputIIDM.xml:34]: same question as above

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling_diagrams/outputIIDM.xml (hunk 2 above)
Comment (rosiereflo) [reply tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling_diagrams/outputIIDM.xml:34]: this is already there in the master but not detected by the comparison script

**Comment by rosiereflo** [comment]
The conflict has been solved

**Pull request merged**